core/src/main/java/org/apache/iceberg/MetadataColumns.java

```diff
@@ -49,26 +54,40 @@ private MetadataColumns() {
   public static final String DELETE_FILE_ROW_DOC = "Deleted row values";
 
   private static final Map<String, NestedField> META_COLUMNS = ImmutableMap.of(
+      SPEC.name(), SPEC,
```

Comment:
nit: should add as the last one instead of the first one (although I know order doesn't matter in a map, better to be consistent in code)

Reply:
I'll switch. Wanted to reduce the number of modified lines but probably better to follow the order of definition.

Reply:
Fixed.